test/openscap_parser_test.rb

```diff
@@ -24,7 +24,7 @@ def setup
         'Red Hat Corporate Profile for Certified Cloud Providers (RH CCP)'
       }
       @report_parser = ::OpenscapParser::Base.new(fake_report)
-      assert_equal 1, @report_parser.profiles.count
+      assert_equal 1, @report_parser.test_result_profiles.count
```

Comment:
If you want profiles associated with the test result only, you've got to ask for it. :)